### DISCUSSION TITLE
delete the useless code

### DISCUSSION THREAD

**Discussion by BRTSG-FOSS** [Bridgetek/IDM2040-7A-EW2022#18] [General]
![image](https://user-images.githubusercontent.com/83748190/176131575-a1512265-fb4b-4585-b054-c2b90444f3d7.png)

https://github.com/Bridgetek/IDM2040-7A-EW2022/blob/main/LDS_Demo/code.py

**Comment by BRTSG-FOSS** [comment]
https://github.com/Bridgetek/IDM2040-7A-EW2022/blob/9e3f611bcc3a06a947f1444f19e1e2b99b47cbd5/LDS_Demo/ui_lds/ui_lds_Details.py#L117

**Reply by jackqfzhang** [reply]
good feedback

**Comment by BRTSG-FOSS** [comment]
https://github.com/Bridgetek/IDM2040-7A-EW2022/blob/9e3f611bcc3a06a947f1444f19e1e2b99b47cbd5/LDS_Demo/ui_lds/ui_lds_Details.py#L41

**Comment by BRTSG-FOSS** [comment]
![image](https://user-images.githubusercontent.com/83748190/176134736-03e3fe08-485b-4194-8317-4491a3a1c7f8.png)


This code.py is not in use any more.
